### DISCUSSION TITLE
[Question] Wezterm keep adding `\` when I copy url from the internet

### DISCUSSION THREAD

**Discussion by haunt98** [wezterm/wezterm#4004] [Q&A] [answered]
For example with url "https://www.youtube.com/watch?v=FCJ3m089p9Q"

After paste, it becomes `https://www.youtube.com/watch\?v\=FCJ3m089p9Q`

<img width="697" alt="image" src="https://github.com/wez/wezterm/assets/5424272/cfa06828-4f88-474c-a34e-ce92838e9b7b">

I use latest wezterm version on macOS `wezterm 20230712-072601-f4abf8fd`

**Comment by wez** [comment]
Are you copying and pasting or dragging and dropping?
It looks like something has quoted the shell special characters in the URL.

wezterm doesn't do that when you paste, in which case it may be something you have configured in your shell.

If you are dragging and dropping then https://wezfurlong.org/wezterm/config/lua/config/quote_dropped_files.html affects how the dropped path is passed to the terminal.

**Reply by haunt98** [reply]
Thanks for your suggestion about the shell (I use omz), will take a look.

**Comment by haunt98** [comment, marked as answer]
Looks lie omz is causing this https://github.com/ohmyzsh/ohmyzsh/issues/7632

After adding `DISABLE_MAGIC_FUNCTIONS=true` to my zshrc, it works.
